pages/profilepage.html

```diff
@@ -16,33 +16,48 @@
     rel="stylesheet"
     href="https://unpkg.com/swiper@7/swiper-bundle.min.css"
     />
+    <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+
     <script type="text/javascript" defer src="../assets/vendor/js/profile_page.js"></script>
     <title>Profile-Page</title>
 </head>
+<script>
+function profile(){
+$.post("../../backend/profile.php", {
+				id:$_SESSION['userid'],
+				email: $_SESSION['email'],
+				key: "gjhtiidsimi09403jfjkdknf"
+			},
+			function(data, status){
+                alert(data);
+				
+			});}
+
+</script>
 <body>
     <div id="profile-page" class="container">
         <div class="content">
             <header class="nav">
                 <!-- should add a nav across all pages to connect everything - for now adding space -->
             </header>
             <div>
-                <h1 class="page-title">Profile Page</h1>
+                <h1 class="page-title"></h1>
             </div>
             <div class="section">
                 <div class="user-details">
                     <div class="profile-pic-container">
                         <img src="../assets/img/anonymous_image.png" alt="Anonymous Profile Pic" class="profile-pic">
 
                     </div>
-                    <p class="username">Jill Anderson</p>
-                    <p class="user-title">UI Designer</p>
-                    <p class="user-description"><i class="fa fa-quote-left quote quote-left" aria-hidden="true"></i>Spend my days designing beautiful websites and applications with a focus on User Experience. Execute on the full spectrum, from ideation to mock-ups to coding to user testing. Live by the words: you can't use up creativity.<i class="fa fa-quote-right quote quote-right" aria-hidden="true"></i></p>
+                    <p class="section-content" id="nameid"></p>
+                    <p class="user-title" id="designation">UI Designer</p>
+                    <p class="user-description" id="intro"><i class="fa fa-quote-left quote quote-left" aria-hidden="true"></i><i class="fa fa-quote-right quote quote-right" aria-hidden="true"></i></p>
```

Comment:
from your usage of "innerHTML" in the profile_page.js, You would be overriding the quotation marks.
Something like this can be the simplest solution.

`<p> <i> <span id="intro"> </span> </i></p>`

Reply:
sir can you please give more explanation on this as i did not get it.